sources/packages/api/src/services/student-account-applications/student-account-applications.models.ts

```diff
@@ -23,3 +23,10 @@ export type StudentAccountApplicationApprovalModel = StudentInfo &
     dateOfBirth: string;
     gender: string;
   };
+
+/**
+ * Data needed to get SIN consent of student
```

Comment:
Can you please change the comment to something like below, please.
```ts
/**
* Dynamic data submitted by the student for the Ministry review while
* requesting his identity verification.
* The properties mapped here represent only the data that must be
* consumed by the API, for instance, the sinConsent.
```

Reply:
Updated.